<?xml version="1.0" encoding="UTF-8"?>
<launch>

	<rosparam command="load" file="$(find city_real)/config/interface_params.yaml" />

	<node pkg="city_real" name="camera_interface" type="camera_interface.py" output="screen"></node>

</launch>